### PR TITLE
[17.0][MIG] purchase_order_type

### PR DESCRIPTION
**Pull request by celm1990**

### CODE REVIEW

**Comment by dalonsod** [comment]
Hello @celm1990 could you add fix commit added by #2136? Thanks!!

**Comment by celm1990** [comment]
> Hello @celm1990 could you add fix commit added by #2136? Thanks!!

@dalonsod  Cherry-pick done, please update your review

File: purchase_order_type/views/view_purchase_order_type.xml
Comment (dalonsod) [inline purchase_order_type/views/view_purchase_order_type.xml]: This column is actually shown. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0 for more details, but code should be modified like this (not tested):
```suggestion
                <field name="active" column_invisible="True" />
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Functional review :+1:  See comment

File: purchase_order_type/views/view_purchase_order_type.xml
Comment (celm1990) [reply purchase_order_type/views/view_purchase_order_type.xml]: Done, thanks for observation

**Review by dalonsod — APPROVED** [review]

**Comment by celm1990** [comment]
@OCA/procurement-purchase-maintainers, could you please help me review this PR and merge it if it is OK?

**Comment by celm1990** [comment]
@pedrobaeza could you help me with merge if it's OK

**Comment by pedrobaeza** [comment]
It requires a review from a PSC.

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**
